src/dipdup/models/__init__.py

```diff
@@ -536,6 +540,9 @@ def __init_subclass__(cls) -> None:
         cls._misses = 0
         cls._cache = LRU(cls._maxsize)
         super().__init_subclass__()
+    
+    def __repr__(self) -> str:
+        return f'<Class {self.__class__.__name__}(hits={self._hits}, misses={self._misses}, maxsize={self._maxsize})>'
```

Comment:
There is a `metrics` section in the crash report with all information about caches. Please, remove this definition.